### PR TITLE
Add Elixir script to bump ex_doc version from CLI

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
This builds on top of #65 so that a pull request can be created to complement the issue.

### CODE REVIEW

File: up_ex_doc_version.exs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+defmodule UpExDocVersion do
+  @mix_exs "mix.exs"
+
+  def up(new_vsn) do
+    content_up =
+      @mix_exs
+      |> File.read!()
+      |> up(new_vsn)
+
+    File.write!(@mix_exs, "#{content_up}\n")
```
Comment (starbelly) [inline up_ex_doc_version.exs:10]: We probably want to call `deps.get` here or remind the user to.

File: mix.exs [hunk 2]
```diff
@@ -1,29 +1,14 @@
...
+  @ex_doc_version "0.30.5"
   def project do
     [
-      app: :rebar_ex_doc,
+      app: :rebar3_ex_doc,
       version: "0.2.20",
       elixir: "~> 1.13",
-      start_permanent: Mix.env() == :prod,
-      deps: deps(),
+      deps: [ex_doc: "~> #{@ex_doc_version}"],
```
Comment (starbelly) [inline mix.exs:9]: The user of the `~>` op here doesn't make sense and that's my bad 😄 We should pin to the exact version.

File: up_ex_doc_version.exs (hunk 1 above)
Comment (paulo-ferraz-oliveira) [reply up_ex_doc_version.exs:10]: 👍 makes sense.

Comment (paulo-ferraz-oliveira) [reply up_ex_doc_version.exs:10]: 15024bd, but it's now a mix task 👍

File: mix.exs (hunk 2 above)
Comment (paulo-ferraz-oliveira) [reply mix.exs]: 3d5d00a

**Comment by paulo-ferraz-oliveira** [comment]
@starbelly, lemme know if the "task" version is an acceptable one. I believe this will imply compiling the code beforehand, but that can be done in the CI that'll invoke this.

On the other hand, I couldn't have `Mix.Task.run` execute on the code (for lack of application start) or `System.cmd` execute on the code (for lack of a proper execution path).

File: mix.lock [hunk 3]
```diff
@@ -1,5 +1,5 @@
 %{
-  "earmark_parser": {:hex, :earmark_parser, "1.4.33", "3c3fd9673bb5dcc9edc28dd90f50c87ce506d1f71b70e3de69aa8154bc695d44", [:mix], [], "hexpm", "2d526833729b59b9fdb85785078697c72ac5e5066350663e5be6a1182da61b8f"},
+  "earmark_parser": {:hex, :earmark_parser, "1.4.36", "487ea8ef9bdc659f085e6e654f3c3feea1d36ac3943edf9d2ef6c98de9174c13", [:mix], [], "hexpm", "a524e395634bdcf60a616efe77fd79561bec2e930d8b82745df06ab4e844400a"},
```
Comment (starbelly) [inline mix.lock:2]: This looks not quite intentional. I think we should upgrade this dep, but not in this PR 😄

Comment (paulo-ferraz-oliveira) [reply mix.lock:2]: This is most probably because just running `mix unlock + deps.get` yielded that change 👍

Comment (paulo-ferraz-oliveira) [reply mix.lock]: ab60185

File: up_ex_doc_version.exs (hunk 1 above)
Comment (paulo-ferraz-oliveira) [reply up_ex_doc_version.exs]: 15024bd, but it's now a mix task 👍

**Comment by starbelly** [comment]
This LGTM!

**Pull request merged**